velociraptor/autoplotter/objects.py

```diff
@@ -1123,14 +1117,18 @@ def parse_yaml(self):
 
         return
 
-    def link_catalogue(self, catalogue: VelociraptorCatalogue):
+    def link_catalogue(self, catalogue: VelociraptorCatalogue, global_mask_tag: str):
         """
         Links a catalogue with this object so that the plots
         can actually be created.
         """
 
         self.catalogue = catalogue
 
+        if global_mask_tag is not None:
```

Comment:
Here you are checking `global_mask_tag` against `None`, but in the function signature you say that  `global_mask_tag` can only be a `str` object.

Reply:
good point, replacing with `Union` typing to allow either